code/index.html

```diff
@@ -3,28 +3,210 @@
+              </div>
+              <div class="checkbox-child">
+              <input 
+                id="featuresChoice" 
+                type="checkbox"
+                name="form"
+              />
+            <label for="featuresChoice">A form</label>
+            </div>
+            <div class="checkbox-child">
+            <input 
+              id="featuresChoice" 
+              type="checkbox"
+              name="articles"
+            />
+            <label for="featuresChoice">Articles</label>
+            </div>
+            <div class="checkbox-child">
+            <input 
+              id="featuresChoice" 
+              type="checkbox"
+              name="links"
+            />
+            <label for="featuresChoice">Links</label>
+            </div>
+            <div class="checkbox-child">
+            <input 
+              id="featuresChoice" 
+              type="checkbox"
+              name="video"
+            />
+            <label for="featuresChoice">Video</label>
+            </div>
+            <div class="checkbox-child">
+            <input 
+              id="featuresChoice" 
+              type="checkbox"
+              name="accordion"
+            />
+            <label for="featuresChoice">Accordion</label>
+            </div>
+          </div>
+          <!--1 extra checkbox-->
+          <div class="form-div checkbox">
+            <div class="checkbox-child">
+            <p>I do care about your security!</p>
+            <input 
+              id="termsconditions" 
+              type="checkbox"
+              name="agree"
+              required
+            />
+            <label for="termsconditions">I agree with the <a href="#">terms</a> and <a href="#">conditions.</a></label>
+            </div>
+          <!-- 1 x Submit button -->
+          <div class="form-div">
+            <div class="submitbtn">
+            <button type="submit">Send form!</button>
+          </div>
+          </div>
+        </form>
+      </div>
+    </section>
 
-      <div class="question" id="section2">Section 2</div>
-      <div class="answer">
-        <p>Lorem ipsum...</p>
+    <section class="accordion">  <!-- Accordion -->
+      <div class="FAQ">
+        <h2>FAQ</h2>
+      </div>
+      <div class="accordion-container-1">
+      <h4 id="question-1" class="question"> Question #1: Do you have a portfolio with previous works? </h4>
+      <div class="answer-container">
+        <p class="answer-text">YES, of course. Please see them <a href="https://github.com/Gabbi-89" target="_blank">here!</a></p>
+      </div>
+      </div>
+      <div class="accordion-container-2">
+      <h4 id="question-2" class="question">Question #2: How much does it cost to get your help? </h4>
+      <div class="answer-container">
+        <p class="answer-text">Please see the pricelist <a href="#">here!</a> Please note that the prices can vary about based on project scoop.</p>
+      </div>
```

Comment:
Missing some indentation which you did so good here above 😢 This is a bit hard to read unfortunately.

Reply:
Thanks for noticing - I fixed this :)